### ISSUE TITLE
Markdown lint (mdl) unclear output

### ISSUE THREAD

**Issue by NightWolf007** [sds/overcommit#678]
.overcommit.yml
```yml
PreCommit:
  Mdl:
    enabled: true
```

README.md
```markdown
## test
```

Overcommit will fail with unclear message:
```
Running pre-commit hooks
Analyze markdown files with mdl.................................[Mdl] FAILED
Errors:
#<Overcommit::Hook::Message:0x00005637bca65c80>
```

`mdl --json README.md` will show:
```json
[{"filename":"README.md","line":1,"rule":"MD002","aliases":["first-header-h1"],"description":"First header should be a top level header"}]
```

**Comment by NightWolf007** [comment]
I see the problem in parsing json response in mdl.rb:
https://github.com/sds/overcommit/blob/master/lib/overcommit/hook/pre_commit/mdl.rb#L22
`JSON.parse` returns a hash with string keys, but there is access to hash by symbol.

I will fix it if you don't mind.

**Comment by sds** [comment]
Fixed in #678. Thanks!
